kitsune/messages/jinja2/messages/inbox.html

```diff
@@ -37,7 +37,11 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
                     {{ avatar_link(message.sender, default_avatar) }}
                 </div>
                 <div class="email-cell from">
+                  {% if message.sender.username == 'SumoBot' %}
```

Comment:
I would move the logic in the view and return a flag like `is_bot_profile` or something similar. The template could be just a boolean check.

Reply:
It looks like you can also pass a name to the `name_link` macro which can be helpful in this case or you can update the macros used to incorporate this logic

Reply:
I think catching it here in the main templates vs. the macro is best - the macro sets up links to user profiles, and we don't want to do that with `deleted user`. Moving the logic there just pollutes that macro, IMHO.

I'll look at sending a flag from the view vs. using the username here.